api/controllers/Cid_EntryController.js

```diff
@@ -26,6 +26,11 @@ module.exports = {
       }
     ),
 
+  prebookingProcess: movements =>
```

Comment:
the other controllers use the term and generally do more, the only thing that happens in this method is destroying the existing prebookings